### ISSUE TITLE
Wiimote Motion Plus not detected

### ISSUE THREAD

**Issue by Wanchai290** [AndersMalmgren/FreePIE#189]
Hey there, I'm still not used to Github so excuse me if I do some things wrong.

I'm using FreePIE 1.11.724 on Windows 10 with a 64-bit machine, and I'm having trouble with the usage of the Wiimote's (RVL-CNT-01) Motion Plus extension with FreePIE
The button input is detected but there's only the gyroscopic data that I can't get, simply because FreePIE thinks there isn't one.
I've tried on a totally different computer, same problem.

I used this code to check if it detected it, as when I tried to read the gyro data, it was always 0
```py
if starting:
	diagnostics.debug(wiimote[0].capabilities.has_available_capability(WiimoteCapabilities.MotionPlus))
	diagnostics.debug(wiimote[0].capabilities.has_enabled_capability(WiimoteCapabilities.MotionPlus))
	
	wiimote[0].enable(WiimoteCapabilities.MotionPlus)

	diagnostics.debug(wiimote[0].capabilities.has_available_capability(WiimoteCapabilities.MotionPlus))
	diagnostics.debug(wiimote[0].capabilities.has_enabled_capability(WiimoteCapabilities.MotionPlus))
```

It is similar to an issue that was posted here, and since it didn't get any fix I'm posting this again as I need it.

**Comment by AndersMalmgren** [comment]
I have lost all my wiimotes when I moved to my new place plus I don't really have time for freepie anymore. I know other dev's lurk around here that have access to said hardware though and they might be able to help. 

Good luck!

**Comment by AndersMalmgren** [comment]
Btw we use dolphins wiimote code albait heavy modified, does everything work in dolphin?

**Comment by Wanchai290** [comment]
Yes my brother used these Wiimotes with Dolphin with their Motion Plus extension and it was perfectly working, thanks for your quick input !
